core/block_svg.js

```diff
@@ -792,6 +792,28 @@ Blockly.BlockSvg.prototype.showContextMenu_ = function(e) {
     if (this.isEditable() && this.workspace.options.comments) {
       menuOptions.push(Blockly.ContextMenu.blockCommentOption(block));
     }
+
+    // pxt-blockly: only allow collapse on top-level event handler blocks
+    if (this.workspace.options.collapse
+        && block.previousConnection == null && block.outputConnection == null) {
+      // Option to collapse/expand block.
+      if (this.collapsed_) {
+        var expandOption = {enabled: true};
+        expandOption.text = Blockly.Msg['EXPAND_BLOCK'];
```

Comment:
we need to localize those strings.

Reply:
these strings were already in the Blockly.Msg dictionary, and it looks like they've been localized for most languages

Reply:
We do not use the blockly localized strings — there is a place (maybe in pxt) where we redefined those using the LF macro.

Reply:
just checked and they've already been localized in pxt as well--it seems like we've localized all the context menu items (like "Inline Inputs" and "External Inputs" are also localized) so that's nice!